spec.bs

```diff
@@ -785,57 +879,81 @@ Issue: This would ideally be replaced by a more descriptive algorithm in Infra.
 <a href="https://github.com/whatwg/infra/issues/201">infra/201</a>
 
 <div algorithm>
+To <dfn>report result</dfn> given a [=leading bid info=] |leadingBidInfo|:
+
+  1. Let |browserSignals| be an [=ordered map=] whose [=map/keys=] are [=strings=] and whose
+    [=map/values=] are {{any}}.
+  1. [=map/Set=] |browserSignals|["`topWindowHostname`"] to [=this=]'s
+    [=relevant settings object=]'s [=environment/top-level origin=]'s [=origin/host=].
+  1. Let |winner| be |leadingBidInfo|'s [=leading bid info/leading bid=].
+  1. [=map/Set=] |browserSignals|["`interestGroupOwner`"] to |winner|'s
+    [=generated bid/interest group=]'s [=interest group/owner=].
+  1. [=map/Set=] |browserSignals|["`renderUrl`"] to |winner|'s [=generated bid/ad descriptor=]'s
+    [=ad descriptor/url=].
+  1. [=map/Set=] |browserSignals|["`bid`"] to |winner|'s [=generated bid/bid=].
+  1. [=map/Set=] |browserSignals|["`desirability`"] to |leadingBidInfo|'s
+    [=leading bid info/top score=].
+  1. [=map/Set=] |browserSignals|["`highestScoringOtherBid`"] to |leadingBidInfo|'s
+    [=leading bid info/highest scoring other bid=].
+  1. TODO: if trusted scoring signals response data version is null, set
+    |browserSignals|["`dataVersion`"].
+  1. If |leadingBidInfo|'s [=leading bid info/top level seller=] is not null, [=map/Set=]
+    |browserSignals|["`topLevelSeller`"] to it.
+  1. If |winner|'s [=generated bid/modified bid=] is not null, [=map/Set=]
+    |browserSignals|["`modifiedBid`"] to it.
+  1. If |leadingBidInfo|'s [=leading bid info/top level seller signals=] is not null, [=map/Set=]
+    |browserSignals|["`topLevelSellerSignals`"] to it.
+  1. If |leadingBidInfo|'s [=leading bid info/component seller=] is not null, [=map/Set=]

```

Comment:
I don't think [=leading bid info/component seller=] is ever set; we need to fix that.